alf/optimizers/optimizers.py

```diff
@@ -380,6 +428,10 @@ def add_param_group(self, param_group):
         else:
             super(NewCls, self).add_param_group(param_group)
 
+        capacity_ratio = self._capacity_ratio
```

Comment:
capacity_ratio should be from scheduler.

Reply:
Changed the name to initial_capacity_ratio. The reason is that we need to record initial capacity ratio in order to be used in ``add_param_group``, since  the capacity scheduler (``self._capacity_ratio``) is inaccessible in ``add_param_group``.